nilearn/datasets/atlas.py

```diff
@@ -0,0 +1,577 @@
+# *- encoding: utf-8 -*-
```

Comment:
Python complained without them. Now solved by replacing all the special characters.